### PR TITLE
Change autocomplete process

### PR DESCRIPTION
**Pull request by martypowell**
The process changes to the following recommendation by GIS

Suggest => Find Address Candidates => Get Schedule

Because there was another api call added, i cleaned up the startup file. this logic should be considered for the Config Javascript Utility, it seems to make it easier to organize.

### CODE REVIEW

File: src/Startup.js [hunk 1]
```diff
@@ -2,43 +2,53 @@ import { Config } from "@baltimorecounty/javascript-utilities";
...
-  },
-  staging: {
-    apiRoot: testApiRoot,
-    addressLookupEndpoint: testAddressLookup,
-  },
-  production: {
-    apiRoot: prodApiRoot,
-    addressLookupEndpoint: `https://services.baltimorecountymd.gov/${addressLookupEndpoint}`,
-  },
+const buildApiUrls = (endpoint) => {
```
Comment (martypowell) [inline src/Startup.js:12]: add a summary here.

File: src/Startup.js [hunk 2]
```diff
@@ -2,43 +2,53 @@ import { Config } from "@baltimorecounty/javascript-utilities";
...
+
+const urls = {
+  collectionSchedule: buildApiUrls("api/hub/collectionSchedule/schedule"),
+  suggest: buildApiUrls("api/hub/gis/Geocoder/suggest"),
+  findAddressCandidates: buildApiUrls(
+    "api/hub/gis/Geocoder/findAddressCandidates"
+  ),
 };
 
+//TODO: We need to merge urls for shared properties
```
Comment (martypowell) [inline src/Startup.js:36]: this should be removed.

File: src/common/Fetch.js [hunk 3]
```diff
@@ -7,7 +7,10 @@ import "whatwg-fetch";
  * @param {object.endpoint} api endpoint
  * @param {object.path} path path for endpoint (path)
  */
-const Fetch = (key, { endpoint, path }) =>
-  fetch(`${endpoint}/${path}`).then((res) => res.json());
+const Fetch = (key, { endpoint, path, queryString }) =>
+  console.log(endpoint, path, queryString) ||
+  fetch(
```
Comment (tmccoy529) [inline src/common/Fetch.js:12]: did you mean to leave this console.log?

Comment (martypowell) [reply src/common/Fetch.js:12]: absolutely not, thanks.

Comment (martypowell) [reply src/common/Fetch.js:12]: updated

**Pull request merged**